src/frequenz/repo/config/nox/config.py

```diff
@@ -14,6 +14,7 @@
 
 import dataclasses as _dataclasses
 import itertools as _itertools
+from pathlib import Path
```

Comment:
Can you keep the same import style (i.e. `import pathlib as _pathlib`? I don't think we actually want to use this particular style, as it proved to have its own issues, but when we decide to change it we can change all in one go.